docs/immortal-empires/high-elves.md

```diff
@@ -0,0 +1,41 @@
+# High Elves
+
+## Tyrion - Eataine
+
+* **Route I - Defender of Ulthuan**
+  * Prince Tyrion is the Defender of Ulthuan and the protector of its people, a heavy burden entrusted to him by the Phoenix King Finubar. Among the Dark Elves of Naggaroth he is known as the Reaper, to the Goblins of Red-Axe Pass he is Orcbane, and to the north the Norse know him as Mankiller. For two centuries he has stood between the Elves of Ulthuan and their many foes as an unbreakable shield against the darkness.
+  * Destroy factions:
+    * Seducers of Slaanesh, The Dreadfleet, Broken Axe, The Drowned
+  * **Reward**:
+    * _The Best Defence is a Good Offence_: Income from Ports: +50%, +10 Leadership (factionwide) 
```

Comment:
`The Best Defence is a Good Offence` -> income from ports (not offensive), +leadership (sort of offensive). 

The objective flavour text is very military, but the reward flavour text is very economic rather than military. Could align these a bit closer?

Reply:
The reward flavour text actually says:
Not since the Age of Colonisation have the Elves known such prosperity.

But I can change the reward title from The Best Defence is a Good Offence to: "Age of Expansion, The Flourishing of the Ten Kingdoms" "Safety and Sanctity for Ulthuan" or something along those lines.

Reply:
I much prefer either of those suggested titles - they're both more generic. "The best defense is a good offence" really makes me think of a military reward otherwise.

Reply:
Done.